README.md

```diff
@@ -128,7 +128,15 @@ Then start the run
 RUN_PARAMS=benchmark_lander_hill cogment run start_run
 ```
 
-and access the playing interface, by default at <http://localhost:8080>
+Access the playing interface by running
```

Comment:
This shouldn't be needed the default `cogment run dev` and `cogment run start` should start the server on port 8080

Reply:
It doesn't anymore, it runs at the port described in the .env file.

Should I change that port to 8080?

Reply:
The port is 8080

https://github.com/cogment/cogment-verse/blob/main/.env#L27

Reply:
My main point is that this is not the way to run the web_client by default :) the docker environment is the default mode.